### PR TITLE
Revised proposal of "AOA multiplexing similar to Android BT/SPP multiplexing"

### PR DESCRIPTION
**Pull request by shiniwat**
This proposal aims to add AOA multiplexing capability to SdlRouterService.

### CODE REVIEW

**Comment by shiniwat** [comment]
Hi @theresalech, this is revised proposal that takes account of what we discussed at issue #286.
This is ready for review. Thanks.

**Comment by shiniwat** [comment]
I got an internal review, and found an inconsistent logic in specific section. It's been updated now. Thanks.

File: proposals/0095-AOA-multiplexing.md [hunk 1]
```diff
@@ -124,53 +128,30 @@ Extends SdlRouterService so that internal class (RegisteredApp) is aware of Tran
...
-
-SdlRouterService selects either BluetoothTransportWriter or AoaTransportWriter according to the TransportType of the RegisteredApp instance:
+### SdlRouterService uses either MultiplexBluetoothTransport or MultiplexAoaTransport depending on the TransportType
 ```java
 public class SdlRouterService extends Service {
     ...
-    // TransportWriters -- for now BT and USB.
-	private BluetoothTransportWriter bluetoothTransportWriter;
-	private AoaTransportWriter aoaTransportWriter;
+    // underlying transport dependning on the TranportType.
```
Comment (theresalech) [inline proposals/0095-AOA-multiplexing.md]: Please update to "underlying transport depending on the TransportType."

Comment (shiniwat) [reply proposals/0095-AOA-multiplexing.md]: Hi @theresalech, thanks for the review.
Fixed the typo you pointed out. Thanks.

Comment (theresalech) [reply proposals/0095-AOA-multiplexing.md]: @shiniwat it looks like there is still one typo.  "TranportType" should be updated to "TransportType" (add an "s").

**Comment by shiniwat** [comment]
Hi @theresalech, thanks for the review. Fixed the typo, too.

**Pull request merged**